code/modules/power/supermatter/supermatter.dm

```diff
@@ -709,6 +709,8 @@ GLOBAL_DATUM(main_supermatter_engine, /obj/machinery/power/supermatter_crystal)
 	for(var/mob/living/carbon/human/l in view(src, HALLUCINATION_RANGE(power)))
 		// Someone (generally a Psychologist), when looking at the SM
 		// within hallucination range makes it easier to manage.
+		if (isnull(l.mind))
+			continue
 		if(HAS_TRAIT(l.mind, TRAIT_SUPERMATTER_SOOTHER))
```

Comment:
Wouldn't appending the mind check to the front of this if statement work just as well?

Reply:
no, because its checked again later. easier to just check in the beginning